samples/src/Microsoft.Azure.IIoT.App/src/Pages/Applications.razor

```diff
@@ -25,7 +25,7 @@
         @foreach (var application in pagedapplicationList.Results) {
         <tr>
             <td>
-                <a href='application/@application.ApplicationId'>
+                <a href='endpoints/null/@application.ApplicationId'>
```

Comment:
Can the null be changed into something else, e.g. "root", or "all"?

Reply:
done